### PR TITLE
Create pentagon.java

### PR DESCRIPTION
**Pull request by bdirksen**
peep my shape

### CODE REVIEW

**Review by rajanmaghera — APPROVED** [review]
Wow I like this pull request! **_Strong_**

**Review by elskhn — APPROVED** [review]
looks good to me 👍

File: app/src/main/java/com/CMPUT301W22T13/CMPUT301W22T13/pentagon.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package com.CMPUT301W22T13.CMPUT301W22T13;
+
+public class pentagon extends Shape{
+    private int vertices;
+    private int sides;
```
Comment (rajanmaghera) [inline app/src/main/java/com/CMPUT301W22T13/CMPUT301W22T13/pentagon.java:5]: Wow so many attributes

**Review by rajanmaghera — CHANGES_REQUESTED** [review]
Maybe add a diameter attribute?

**Review by rajanmaghera — APPROVED** [review]

**Comment by kakaname** [comment]
Look good

**Pull request merged**